chat-plugins/hangman.js

```diff
@@ -130,33 +130,33 @@ class Hangman extends Rooms.RoomGame {
 			result = 2;
 		}
 
-		let output = '<div class="broadcast-' + (result === 1 ? 'red' : (result === 2 ? 'green' : 'blue')) + '">';
-		output += '<p style="text-align:left;font-weight:bold;font-size:10pt;margin:5px 0 0 15px">' + (result === 1 ? 'Too bad! The mon has been hanged.' : (result === 2 ? 'The word has been guessed. Congratulations!' : 'Hangman')) + '</p>';
-		output += '<table><tr><td style="text-align:center;">' + this.hangingMan() + '</td><td style="text-align:center;width:100%;word-wrap:break-word">';
+		let output = `<div class="broadcast-${result === 1 ? 'red' : (result === 2 ? 'green' : 'blue')}">`;
+		output += `<p style="text-align:left;font-weight:bold;font-size:10pt;margin:5px 0 0 15px">${result === 1 ? 'Too bad! The mon has been hanged.' : (result === 2 ? 'The word has been guessed. Congratulations!' : 'Hangman')}</p>`;
+		output += `<table><tr><td style="text-align:center;">${this.hangingMan()}</td><td style="text-align:center;width:100%;word-wrap:break-word">`;
 
 		let wordString = this.wordSoFar.join('');
 		if (result === 1) {
 			let word = this.word;
 			wordString = wordString.replace(/_+/g, (match, offset) =>
-				'<font color="#7af87a">' + word.substr(offset, match.length) + '</font>'
+				`<font color="#7af87a">${word.substr(offset, match.length)}</font>`
 			);
 		}
 
-		if (this.hint) output += '<div>(Hint: ' + Tools.escapeHTML(this.hint) + ')</div>';
-		output += '<p style="font-weight:bold;font-size:12pt;letter-spacing:3pt">' + wordString + '</p>';
+		if (this.hint) output += `<div>(Hint: ${Tools.escapeHTML(this.hint)})</div>`;
+		output += `<p style="font-weight:bold;font-size:12pt;letter-spacing:3pt">${wordString}</p>`;
 		if (this.guesses.length) {
 			if (this.letterGuesses.length) {
-				output += 'Letters: ' + this.letterGuesses.map(g =>
-					'<strong' + (g[1] === '1' ? '' : ' style="color: #DBA"') + '>' + Tools.escapeHTML(g[0]) + '</strong>'
-				).join(', ');
+				output += `Letters: ${this.letterGuesses.map(g =>
+    '<strong' + (g[1] === '1' ? '' : ' style="color: #DBA"') + '>' + Tools.escapeHTML(g[0]) + '</strong>'
+).join(', ')}`;
```

Comment:
The spacing is off because it's using multiline template strings. Do not use multiline template strings.